packages/launch/src/focused-launch/domain-details/index.tsx

```diff
@@ -36,7 +35,7 @@ const DomainDetails: React.FunctionComponent = () => {
 
 	const launchFlowUrl = getCurrentLaunchFlowUrl();
 	const redirectToUseDomainFlow = (): void => {
-		const useYourDomainUrl = `/start/new-launch/domains-launch/use-your-domain?siteSlug=${ siteSubdomain?.domain }&source=${ launchFlowUrl }`;
+		const useYourDomainUrl = `/start/launch-site/domains-launch/use-your-domain?siteSlug=${ siteSubdomain?.domain }&source=${ launchFlowUrl }`;
```

Comment:
Note to self: Confirm if redirecting to `/start/launch-site/...` is correct and whether `source` is still needed in the query string (not defined in `launch-site` deps).

Reply:
`source` in this case is used to add a link to the Back button of `use-your-domain` screen
<img width="1101" alt="Screenshot 2021-07-22 at 15 58 35" src="https://user-images.githubusercontent.com/14192054/126642915-189918b6-2bb1-4b3c-8999-d392d1e905bf.png">